freespace2/freespace.cpp

```diff
@@ -7376,28 +6981,26 @@ void Do_model_timings_test()
 }
 #endif
 
-// Call this function when you want to inform the player that a feature is disabled in this build
+/// Call this function when you want to inform the player that a feature is disabled in this build
```

Comment:
Why the extra slash?

Reply:
three / is the doxygen short form for 
`/**
* @brief blablabla
*/`